frontend/src/pages/settings/logs-and-activity.vue

```diff
@@ -115,14 +111,14 @@
   </settings-page>
 </template>
 
-<route lang="yaml">
-meta:
-  admin: true
-</route>
```

Comment:
I removed this since it wasn't letting my admin account access the page. I didn't see it on the api keys or devices page which I think are admin only as well. Should these all be removed or is there a more correct way to perform this check?